src/github.com/getlantern/yamlconf/yamlconf.go

```diff
@@ -103,6 +103,14 @@ type Manager struct {
 	// example for fetching config updates from a remote server.
 	CustomPoll func(currentCfg Config) (mutate func(cfg Config) error, waitTime time.Duration, err error)
 
+	// Obfuscate: if true, the on-disk version of the config will be obfuscated by
+	// encrypting it using AES-128 encryption.
```

Comment:
I added some timing to the loadFromDisk method and printed out the results.  I just wrote a benchmark for thoroughness.  The benchmark reads a realistic config file from disk and decodes the yaml.  Depending on the run, the encryption adds anywhere between 0 and 13% overhead.  Without the yaml decoding the differences are much more severe, but the yaml decoding dwarfs the other work.